server/models/Document.ts

```diff
@@ -255,14 +255,18 @@ class Document extends ParanoidModel<
   @Column
   editorVersion: string;
 
-  /** An emoji to use as the document icon. */
+  /**
+   * An emoji to use as the document icon,
+   * This is used as fallback (for backward compat) when icon is not set.
+   */
   @Length({
-    max: 1,
-    msg: `Emoji must be a single character`,
+    max: 50,
```

Comment:
This matches the `icon` validation (double write).

Reply:
We can have a follow up PR to remove everything related to `emoji` column in DB